src/datamessage/DataMessage.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/* eslint @typescript-eslint/no-explicit-any: 0 */
```

Comment:
nit: you can put this line before `json(): any {` since `any` is being used only once